src/test/groovy/FilebeatStepTests.groovy

```diff
@@ -161,7 +161,7 @@ class FilebeatStepTests extends ApmBasePipelineTest {
     )
     printCallStack()
     assertTrue(assertMethodCallContainsPattern('readJSON', "file=${workdir}/${config}"))
-    assertTrue(assertMethodCallContainsPattern('sh', "docker exec ${id}"))
+    assertTrue(assertMethodCallContainsPattern('sh', "docker exec -t ${id}"))
```

Comment:
Fixed, cool!

![image](https://media.giphy.com/media/AvMJCeu1EMmhG/giphy.gif)

Reply:
this is good or bad 😄

Reply:
This should be the reaction to every fix... It's funny, which is good :)